python/orca/tutorial/NCF/pytorch_dataset.py

```diff
@@ -137,8 +146,15 @@ def get_input_dims(users, items, sparse_features, dense_features):
 
 
 def process_ratings(dataset_dir, user_num, item_num):
```

Comment:
this method only needs a single ratings file and only need to fetch one remote file to local instead of the dir?